### DISCUSSION TITLE
Social media links not showing (footer/author)

### DISCUSSION THREAD

**Discussion by xDGameStudios** [mmistakes/minimal-mistakes#2924] [Q&A] [answered]
I'm not able to display (turn on) the social media links both on the footer and authors side bar. This is not a bug per-se as I'm probably missing some config. Yet, I read through all the documentation and was unable to find an answer.

**Comment by iBug** [comment, marked as answer]
Duplicate of #413?

**Comment by xDGameStudios** [comment]
Ohh didn't see that post! Maybe it's something that could be added to the documentation. Either way, I will try it when I get home to see if that is actually the problem.
